qcore/microtime.pyi

```diff
@@ -62,6 +63,10 @@ def format_utime_as_iso_8601(
     utime: Utime, *, drop_subseconds: bool = False, tz: timezone = timezone.utc
 ) -> str: ...
 
+# datetime.fromisoformat() is new in Python 3.7.
+if hasattr(datetime, "fromisoformat"):
```

Comment:
This isn't allowed in stubs, we must use `if sys.version_info >= (3, 7):`

Reply:
Oh, right. Not sure how the checks are passing, though... :/

Btw, would I need to `import sys`, or that's not needed?